### DISCUSSION TITLE
Setting the date range

### DISCUSSION THREAD

**Discussion by andythinkeq** [dmuy/duDatepicker#50] [Q&A]
I've got a problem setting the date range after a post

I use this, but it fails and crashes javascript

duDatepicker('.fld-dates', { 
   range: true, 
    value: '30/04/2023 - 31/05/2023',
    format: 'mmmm d, yyyy' 
});

What am I doing wrong?

**Comment by dmuy** [comment]
Hi @andythinkeq,

It is written [here](https://github.com/dmuy/duDatepicker/wiki/Usage#setting-default-value) on how to set the value on date range mode. I notice your `format` config is `mmmm d, yyyy`, so you should set the value using that format too.

**Comment by andythinkeq** [comment]
Thanks I'll give that a try

**Comment by andythinkeq** [comment]
I'm still having problems, can anyone see what the issue may be?

OK, for clarity I've used dates where the dd and mm can't be confused as we don't have a 13th or 29th month

I chose this date range
13/10/2023 - 29/11/2023

Posted the form and the code I ran to set the dates put this in the visual field;
13/10/2023-29/11/2023
using the code below

jQuery(document).ready(function($){

    duDatepicker('.fld-dates', {
        range: true,
        value: '13/10/2023-29/11/2023',
    });

});   

But the input element is set to
<input type="text" name="f1-dates" id="f1-dates" class="fld-dates form-control" placeholder="Between Dates" value="13/10/2023-29/11/2023" data-range-from="01/10/2024" data-range-to="05/11/2025">

What am I doing wrong?

**Reply by dmuy** [reply]
Hi @andythinkeq,

You should set the format to `dd/mm/yyyy` because the default format is `mm/dd/yyyy`. Like so:
```javascript
duDatepicker('.fld-dates', {
    format: 'dd/mm/yyyy', range: true,
    value: '13/10/2023-29/11/2023',
});
```
